apps/charterafrica/src/payload/collections/Explainers.js

```diff
@@ -11,14 +11,34 @@ const Explainers = {
   },
   fields: [
     {
-      name: "title",
+      type: "collapsible",
       label: {
         en: "Title",
         fr: "Titre",
         pt: "Título",
       },
-      type: "text",
-      localized: true,
+      fields: [
+        {
+          name: "title",
+          label: {
+            en: "Title",
+            fr: "Titre",
+            pt: "Título",
+          },
+          type: "text",
+          required: true,
```

Comment:
@kilemensi Would it still be necessary to have this as required?

Reply:
Without titles, 👇🏽 is how explainers look in the CMS.

<img width="807" alt="Screenshot 2023-03-17 at 11 37 15" src="https://user-images.githubusercontent.com/1779590/225854489-fe887f11-1daf-4f28-96cd-06751a6c6820.png">